asv/plugins/conda.py

```diff
@@ -85,30 +86,44 @@ def _matches(cls, python):
 
     def _setup(self):
         try:
-            conda = util.which('conda')
+            conda = self.find_executable('conda')
```

Comment:
This must use `util.which`.
The environment is not initialized yet at this stage, so it should not try looking for the conda executable inside it.